src/Common/src/CoreLib/System/Collections/CompatibleComparer.cs

```diff
@@ -17,10 +17,6 @@ internal CompatibleComparer(IHashCodeProvider hashCodeProvider, IComparer compar
             _comparer = comparer;
         }
 
-        internal IHashCodeProvider HashCodeProvider => _hcp;
-
-        internal IComparer Comparer => _comparer;
-
```

Comment:
These need to be put back. This code is shared with CoreLib in the coreclr repo, and these properties are used there.

Reply:
My mistake, should be back now.